### ISSUE TITLE
Dot net core 1

### ISSUE THREAD

**Issue by SteveStrong** [domaindrivendev/Swashbuckle#768]
Have anyone verified this works with .net core rc2?

**Comment by ascolieri** [comment]
I think it needs to update the dependencies to use the new asp.net core ones

**Comment by ilessa** [comment]
Looks as though another issue #770 been opened for this as well

**Comment by SteveStrong** [comment]
so I just tried a recompile on asp.net core RC2 and there are a lot of missing API's, it looks like more that just changing namespaces, functionality has moved.  I am willing to help, is there a branch that is close?

**Comment by xperiandri** [comment]
Same question

**Comment by domaindrivendev** [comment]
The work for vnext/core/rc2 etc. is currently being done in a separate repo (confusing I know) - https://github.com/domaindrivendev/Ahoy. See milestone 6.0.0-beta2 for an rc2 comparable release. Will be merging back into this repo at some point

**Comment by gencebay** [comment]
Hi, I have released rc2-final compatible Swashbuckle
https://github.com/gencebay/Ahoy/releases/tag/6.0.0-rc2-final 
@domaindrivendev can we merge them both?

**Comment by xperiandri** [comment]
Cool!

**Comment by markvincze** [comment]
hi @gencebay, (cc @domaindrivendev)

Cool stuff, thanks for adjusting it this quickly!

Do you guys know when the new version of `Swashbuckle.SwaggerGen` and `Swashbuckle.SwaggerUi` will be pushed to Nuget?

Thanks!
Mark

**Comment by rfcdejong** [comment]
It would indeed be very nice to be able to use the latest branch by modifying the project.json file.

```
    "Swashbuckle": "6.0.0-rc2-final",
    "Swashbuckle.SwaggerGen": "6.0.0-rc2-final",
    "Swashbuckle.SwaggerUi": "6.0.0-rc2-final",
```

**Comment by pherbel** [comment]
Hi,

@gencebay Was your rc2 package published to nuget? or myget?

:+1:

**Comment by gencebay** [comment]
@pherbel not yet! waiting for @domaindrivendev  review, and I dont want to create duplicate packages but also you can fetch the latest repo then build and test on your local machine.

**Comment by rfcdejong** [comment]
The following topic helped me for now, I cloned the package and now the source locally is used as a NuGet reference.  http://weblog.west-wind.com/posts/2015/Jun/09/Using-and-Debugging-External-Source-Code-Packages-in-ASPNET-5

**Comment by domaindrivendev** [comment]
This is available with the 6.0.0-beta9 Nuget package. NOTE: The source code for 6.0.0 currently resides at https://github.com/domaindrivendev/Ahoy
